_docs/language-support.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Language Support
+permalink: /docs/user-tutorials/language-support/
+---
+
+Ideally, all features in Dockstore would be available in all languages. 
+However, due to time constraints and gaps in our knowledge of different workflows, some features of Dockstore are not available in all languages. 
+
+To help lay out what parts of Dockstore are available in which languages, we cover the following guide for what features are available on the Dockstore site and in the Dockstore command-line utility. 
+
+{% include language-support.html %}
+
+<sup>[1] Some verified tools/workflows are known to execute unsuccessfully with Bunny.  As a result, Bunny through Dockstore will also execute unsuccessfully.
```

Comment:
"As a result, Bunny through Dockstore will also execute unsuccessfully" can be misread to mean always. Maybe re-phrase to indicate that only those specific tools/workflows or workflows that are similar would fail.